server/channels/store/localcachelayer/channel_layer.go

```diff
@@ -225,6 +225,39 @@ func (s LocalCacheChannelStore) SaveMultipleMembers(members []*model.ChannelMemb
 	return members, nil
 }
 
+func (s LocalCacheChannelStore) GetChannelsMemberCount(channelIDs []string, allowFromCache bool) (_ map[string]int64, err error) {
+	if !allowFromCache {
+		return s.ChannelStore.GetChannelsMemberCount(channelIDs, false)
+	}
+
+	counts := make(map[string]int64)
+	remainingChannels := make([]string, 0)
+
+	for _, channelID := range channelIDs {
+		var cacheItem int64
+		err := s.rootStore.doStandardReadCache(s.rootStore.channelMemberCountsCache, channelID, &cacheItem)
+		if err == nil {
+			counts[channelID] = cacheItem
+		} else {
+			remainingChannels = append(remainingChannels, channelID)
+		}
+	}
+
+	if len(remainingChannels) > 0 {
+		remainingChannels, err := s.ChannelStore.GetChannelsMemberCount(remainingChannels, false)
+		if err != nil {
+			return nil, err
+		}
+
+		for remainingChannelID, remainingChannelCount := range remainingChannels {
```

Comment:
Rename remainingChannelID and remainingChannelCount to `id` and `count` respectively.